service/src/main/java/uk/gov/hmcts/reform/fpl/service/document/ManageDocumentLAService.java

```diff
@@ -63,64 +52,13 @@ public Map<String, Object> baseEventData(CaseData caseData) {
 
         if (isNotEmpty(caseData.getAllRespondents())) {
             listAndLabel.put(RESPONDENTS_LIST_KEY, caseData.buildRespondentDynamicList());
+            listAndLabel.put(RESPONDENT_LIST_KEY, caseData.buildRespondentDynamicList());
```

Comment:
can we maybe rename to make clear that these are for different flows - furtherevidence and position statement